### ISSUE TITLE
Some questions about the paper

### ISSUE THREAD

**Issue by handsomelys** [invictus717/MiCo#3]
How do you understand $`E_{Sam}`$ and the corresponding $`E^{T-I}_{Sam}`$ in the paper? Is it constructed using the positional embedding in the transformer like the learnable embedding $`E_{Pos}`$ etc. mentioned above?

**Comment by invictus717** [comment]
Exactly. They are just random initialized vanilla positional embeddings.

**Comment by invictus717** [comment]
These paired embeddings share the same weights to label the corresponding text paired datasets

**Comment by handsomelys** [comment]
> These paired embeddings share the same weights to label the corresponding text paired datasets

Thanks Reply!

**Comment by handsomelys** [comment]
Thank you very much for your enthusiastic reply. But I still have some questions about pretraining objectives: 
Are the two terms in Formula 4 not equivalent? Why?

![image](https://github.com/invictus717/MiCo/assets/48092352/fd051ae8-1d13-4f9b-9452-adde4645fb0e)

What is the value of predictions $`p_v`$ in Formula 5? Is the value of $`p_v`$ obtained directly through an MLP layer or will it go through an activation function similar to sigmoid?

![image](https://github.com/invictus717/MiCo/assets/48092352/c88ced9c-2c76-4c6c-8741-c56b055b38a6)

How is the conditional causal masked mentioned in Formula 6 done? Is it to mask the last 60% of all tokens, and then use BERT to reconstruct the masked tokens in an autoregressive manner ?

![image](https://github.com/invictus717/MiCo/assets/48092352/00437cc6-ecd3-472c-b0a9-ab5a0c4ce327)

**Comment by invictus717** [comment]
The match process is provided in our released code:

https://github.com/invictus717/MiCo/blob/89c91c9dac68125a18a1a966bd80f9e74e584e80/model/mico.py#L44

**Comment by invictus717** [comment]
The causal pretraining process is exactly as you say, which is intuitive and simple.

**Comment by invictus717** [comment]
If you have any further questions, please feel free to reach out.

**Comment by gongbudaizhe** [comment]
Hi, may I ask some questions about the paper?

1. Why Z_I = [Z_I^1, Z_I^2, ... Z_I^LI]? what does LI mean? what is sample length? is it the num frames of a video?
2. What does E_C^I mean? Why we need this term?
